cocoapods-plugin/lib/cocoapods-xcremotecache/command/hooks.rb

```diff
@@ -27,8 +27,9 @@ module Hooks
     LLDB_INIT_COMMENT="#RemoteCacheCustomSourceMap"
     LLDB_INIT_PATH = "#{ENV['HOME']}/.lldbinit"
     FAT_ARCHIVE_NAME_INFIX = 'arm64-x86_64'
+    XCRC_COOCAPODS_ROOT_KEY = 'XCRC_COOCAPODS_ROOT'
 
-    # List of plugins' user properties that should be copied to .rcinfo 
+    # List of plugins' user properties that should not be copied to .rcinfo 
```

Comment:
It was a silly mistake in #52: there is `! CUSTOM_CONFIGURATION_KEYS ` in the expression: 
https://github.com/spotify/XCRemoteCache/blob/91505a59d2a3b8db3dc0eef90c6fbdc051141169/cocoapods-plugin/lib/cocoapods-xcremotecache/command/hooks.rb#L96